extensions/amp-accordion/1.0/accordion.js

```diff
@@ -72,8 +81,73 @@ export function Accordion({
     });
   }, [expandSingleSection]);
 
+  useImperativeHandle(
```

Comment:
IMHO, the `const toggle = ` and other declarations should precede this call.

Reply:
Thanks, updated!